<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QWidget" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">gridline-color: rgb(0, 0, 0);
background-color: rgb(70, 70, 70);
alternate-background-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
selection-color: rgb(0, 85, 255);
selection-background-color: rgb(0, 0, 127);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>21</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Interface</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="interfaceBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>21</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
gridline-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="configBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Ip Address</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipBox1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipBox2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipBox3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipBox4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subBox4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subBox3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subBox1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Subnet</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="subBox2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="shutBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
alternate-background-color: rgb(66, 66, 66);
background-color: rgba(70, 70, 70, 70);
gridline-color: rgb(0, 0, 0);
selection-color: rgb(255, 170, 0);
selection-background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Shutdown</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="duplexCombo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>Auto Duplex</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Half Duplex</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Full Duplex</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Duplex</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QComboBox" name="speedCombo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>Auto</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color :rgb(0, 170, 255)</string>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="descripBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="clockCombo">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>56000 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>64000 </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>72000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>125000  </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>128000   </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>148000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>250000   </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>500000  </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>800000   </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000000  </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1300000   </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000000  </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4000000</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="clockCheck">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);
alternate-background-color: rgb(66, 66, 66);
background-color: rgba(70, 70, 70, 70);
gridline-color: rgb(0, 0, 0);
selection-color: rgb(255, 170, 0);
selection-background-color: rgb(66, 66, 66);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Clock Rate</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>interfaceBox</tabstop>
  <tabstop>ipBox1</tabstop>
  <tabstop>ipBox2</tabstop>
  <tabstop>ipBox3</tabstop>
  <tabstop>ipBox4</tabstop>
  <tabstop>subBox1</tabstop>
  <tabstop>subBox2</tabstop>
  <tabstop>subBox3</tabstop>
  <tabstop>subBox4</tabstop>
  <tabstop>shutBox</tabstop>
  <tabstop>duplexCombo</tabstop>
  <tabstop>speedCombo</tabstop>
  <tabstop>clockCheck</tabstop>
  <tabstop>clockCombo</tabstop>
  <tabstop>descripBox</tabstop>
  <tabstop>clearBtn</tabstop>
  <tabstop>configBtn</tabstop>
  <tabstop>exitBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
